Sources/CombineGRPC/Client/RetryPolicy.swift

```diff
@@ -26,7 +26,7 @@ public enum RetryPolicy {
    ```
    */
   case failedCall(upTo: UInt = 1,
-                  when: (GRPCStatus) -> Bool,
+                  when: (RPCError) -> Bool,
```

Comment:
Note that `RetryPolicy` now takes an `RPCError` instead of a `GRPCStatus` in the predicate.